cerbero/utils/shell.py

```diff
@@ -159,8 +159,17 @@ def unpack(filepath, output_dir):
         zf = zipfile.ZipFile(filepath, "r")
         zf.extractall(path=output_dir)
 
+def get_dl_mirror(url, mirror_url, filename = None):
+    if mirror_url is None:
+      return url
+    if filename is None:
+      dl_mirror_url = mirror_url + os.path.basename(url)
+    else:
+      dl_mirror_url = mirror_url + filename
+    logging.info('Using the mirror url: %s instead of %s' % (dl_mirror_url, url))
```

Comment:
Use cerbero's logging system

Reply:
I dont understand, the method above uses logger.info

Reply:
ok